documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -362,6 +367,42 @@ ifdef::community[]
 == Confluent Schema Registry
 
 There is an alternative https://github.com/confluentinc/schema-registry[schema registry] implementation provided by Confluent.
+
+// Type: concept
+// Title: Overview of deploying a Confluent Schema Registry
```

Comment:
These annotation comments are intended for use with content that is exposed downstream and should not be used for community-only content. Unless the Confluent integration is something that we intend to expose in the product, these comments should be removed.

Reply:
done